server/build.rs

```diff
@@ -0,0 +1,21 @@
+//! build script to generate .rs from .proto
+
+///generates .rs files in src directory
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    // TODO Also import common interface files
+    // Build the Client
+    tonic_build::configure()
```

Comment:
Use the method Roman is now using, where the configuration is cloned: https://github.com/Arrow-air/svc-scheduler/blob/roman/dw-384/svc-scheduler-interfaces/server/build.rs#L5